src/app/education/education-card/education-card.component.css

```diff
@@ -0,0 +1,33 @@
+
+::ng-deep .mat-vertical-stepper-content {
+    visibility: visible !important;
+    height: 100% !important;
+  }
+  
+  .step-card{
```

Comment:
Could you pls apply the code style to this file, the code seems to be missaligned?

Reply:
Done